tasks/patterns.js

```diff
@@ -1,29 +1,34 @@
 const downpour = require('@sparkbox/downpour');
+const markdown = require('helper-markdown');
+
 const downpourOptions = {
   src: {
     data: {
-      basedir: 'src/drizzle/data',
-      glob: 'src/drizzle/data/**/*.yaml',
+      basedir: 'src/data',
```

Comment:
The diff here gives a nice view on how I moved some files around.

I moved `data` out of our `drizzle` directory. Now we can rename `drizzle` to something better! I called it `markup`.